phpBB/config/default/container/services_messenger.yml

```diff
@@ -0,0 +1,57 @@
+parameters:
+    core.messenger_queue_file: '%core.cache_dir%queue.%core.php_ext%'
+
+services:
+    messenger.method_collection:
+        class: phpbb\di\service_collection
+        arguments:
+            - '@service_container'
+        tags:
+            - { name: service_collection, tag: messenger.method, class_name_aware: true }
+
+    messenger.method.base:
+        class: phpbb\messenger\method\base
+        shared: false
+        arguments:
+            - '@assets.bag'
+            - '@config'
+            - '@dispatcher'
+            - '@language'
+            - '@log'
+            - '@request'
+            - '@user'
+            - '@messenger.queue'
+            - '@path_helper'
+            - '@ext.manager'
+            - '@template.twig.extensions.collection'
+            - '@template.twig.lexer'
+            - '%core.template.cache_path%'
+            - '%core.php_ext%'
+
+    messenger.method.email:
+        class: phpbb\messenger\method\phpbb_email
+        shared: false
+        parent: messenger.method.base
+        calls:
+            - [reset, []]
+            - [set_transport, []]
```

Comment:
It's used in several places, f.e. sending email and process queue. So I've decided just to initialize it from the start so that it would be the same object for different places.